Description: Add AM_MAINTAINER_MODE to configure.in
 Disable maintainer mode by default, as should be, to avoid having the
 Makefiles try to run autotools during the build.
Author: Julien BLACHE <jblache@debian.org>

Index: libieee1284-0.2.11/configure.in
===================================================================
--- libieee1284-0.2.11.orig/configure.in	2010-05-06 11:24:03.000000000 +0200
+++ libieee1284-0.2.11/configure.in	2010-05-06 11:24:18.000000000 +0200
@@ -5,6 +5,8 @@
 AC_PREREQ(2.53)
 AM_CONFIG_HEADER(include/config.h)
 
+AM_MAINTAINER_MODE
+
 dnl Checks for programs.
 AC_PROG_CC
 AM_PROG_CC_C_O
